src/cozy/services/cozy-client.service.ts

```diff
@@ -17,3 +17,7 @@ export class CozyClientService {
         return StaticCozyClient.client;
     }
 }
+
+export class IsInstalled {
+    static value = false;
+}

```

Comment:
Shouldn't this variable be elsewhere? This is something related to the vault, not the client instance, no? 

Also, what is installed? Does it mean `isVaultCreated`?

Reply:
I forgot to squash this commit, but there is already a different name in the code base. Current is `UserFinishedInstallation`, but to be honest I'm not happy with this name. This represent a state where the user finished the configuration process but did not created the vault yet (as it is created only on the first login). This variable exist only to authorize the user to access the login page (due to the guard).

I can move it elsewhere, I struggled to make it a singleton (so it is shared between guards and code) due to the way services are split into modules. But I should check this part again.

Reply:
`UserFinishedInstallation` is way better ;). Can't we have an `InstanceService` or something? With informations related to... the cozy instance?

Reply:
I moved it back to the VaultInstallationService : https://github.com/cozy/cozy-pass-web/pull/9/commits/87d3ea110990105b3482ee3453390786688f30e0

This exact configuration did not work last week (it was not injected as a singleton in the installation page), this may be related to the jslib conflict we had.